src/Raven.Studio/wwwroot/App/views/database/indexes/editIndex.html

```diff
@@ -346,19 +346,53 @@ <h2 class="margin-top margin-top-sm">No fields added</h2>
                         </div>
                     </div>
                     <div role="tabpanel" class="tab-pane fade" id="archivedData">
-                        <div class="alert alert-info">
-                            HERE GOES DESCRIPTION
+                        <div class="alert alert-info small no-padding-left">
+                            <ul>
+                                <li>
+                                    <div class="margin-bottom-xs">
+                                        The default indexing behavior of archived data depends on the index data source:
+                                    </div>
+                                    <div class="margin-left">
+                                        <table class="table table-condensed">
```

Comment:
@Danielle9897 Can we put that inside col-md-X instead of setting width of 20% via styles? I will be more responsive.

Reply:
done